config/management/operational/src/validate_transaction.rs

```diff
@@ -17,9 +18,8 @@ pub struct ValidateTransaction {
 
 /// Returns `true` if we've passed by the expected sequence number
 impl ValidateTransaction {
-    pub fn execute(self) -> Result<bool, Error> {
-        let client = JsonRpcClientWrapper::new(self.host);
-        let sequence_number = client.sequence_number(self.account_address)?;
-        Ok(sequence_number >= self.sequence_number)
+    pub fn execute(self) -> Result<Option<VMStatusView>, Error> {
+        JsonRpcClientWrapper::new(self.host)
+            .transaction_status(self.account_address, self.sequence_number)
     }
```

Comment:
So this isn't really `ValidateTransaction` anymore, as much as `TransactionStatus`

Reply:
or check transaction so that it implies an action / verb instead of a state or noun

Reply:
I'm kinda inclined that validate is okay :P